src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
I think we still want this, but it's precedence should match the precedence in the parser (I think I commented about this on the original diff).